hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/H2/V1_12_8__change_length_of_created_last_modified_by___H2.sql

```diff
@@ -0,0 +1,45 @@
+ALTER TABLE sp_action ALTER COLUMN created_by VARCHAR(64);
```

Comment:
I think we could alter multiple columns at once using one SQL query:
ALTER TABLE sp_action ALTER COLUMN
created_by SET DATA TYPE VARCHAR(64),
last_modified_by SET DATA TYPE VARCHAR(64);

Not 100% sure if H2 supports that... could you pls check?
Thx